### DISCUSSION TITLE
Performance issues with UTMStack.

### DISCUSSION THREAD

**Discussion by tssingtampa** [utmstack/UTMStack#497] [Q&A] [answered]
Hello, community,

I have UTMStack installed on a PC with the following specifications:
Core i5 4590T at 3 GHz
16 GB RAM
500 GB storage.

I have 8 agents connected and a FORTINET firewall sending logs.

The system is experiencing stability issues; the web console crashes, sometimes doesn't load properly, and when I'm doing reviews, it doesn't respond well.

![image](https://github.com/utmstack/UTMStack/assets/142441903/da49ff03-d712-4d3d-9af8-e23d9d455f49)

**Comment by c3s4rfred** [comment, marked as answer]
Hi @tssingtampa, that error is common when UTMStack docker's Stack is restarting, check if you have something that causes that behavior, if you are familiar with docker can check the state of the stack using docker ps command, and then, you can check the logs of the containers to see if there are any errors.
